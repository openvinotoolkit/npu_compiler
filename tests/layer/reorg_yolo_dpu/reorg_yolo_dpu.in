version https://git-lfs.github.com/spec/v1
oid sha256:89bebd0c9be8362cc119c033452903712d5127447d17f686ad21c70ef517f47d
size 16384
