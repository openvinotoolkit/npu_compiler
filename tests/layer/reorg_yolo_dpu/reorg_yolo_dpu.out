version https://git-lfs.github.com/spec/v1
oid sha256:5bf303f5342bad16ae9e7395e004d92580fb02c393cc6ea8bf1ec037edc7806f
size 16384
